.github/autobuild/ios.sh

```diff
@@ -19,7 +19,14 @@ setup() {
     else
         echo "Install dependencies..."
         python3 -m pip install "aqtinstall==${AQTINSTALL_VERSION}"
-        python3 -m aqt install-qt --outputdir "${QT_DIR}" mac ios "${QT_VERSION}" --archives qtbase qttools qttranslations qtmacextras
+        # Install actual ios Qt:
+        python3 -m aqt install-qt --outputdir "${QT_DIR}" mac ios "${QT_VERSION}" --archives qtbase qttools qttranslations
+        if [[ ! "${QT_VERSION}" =~ 5\..* ]]; then
```

Comment:
Are you using the _QT_ version to check the _target platform_?  That doesn't seem right.  Why not have parameter or separate variable?

Reply:
Nah, it's quite something else: This is all about the ios target platform.
The issue is that Qt5 ships qmake as part of the aqt installation, while Qt6 only ships a shim which redirects to the mac/desktop qmake. Therefore, this is needed additionally for Qt6.

Maybe the comment should be improved?

Reply:
Yes :)

Reply:
Done.